maat-court-data-api/src/main/java/gov/uk/courtdata/billing/controller/ApplicantHistoryBillingController.java

```diff
@@ -0,0 +1,30 @@
+package gov.uk.courtdata.billing.controller;
+
+import gov.uk.courtdata.annotation.StandardApiResponseCodes;
+import gov.uk.courtdata.billing.service.ApplicantHistoryBillingService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@Tag(name = "Applicant History Billing Extract", description = "Rest API for extracting applicant history to send to billing teams.")
+@RequestMapping("${api-endpoints.billing-domain}/applicant-history")
+public class ApplicantHistoryBillingController {
+
+    private final ApplicantHistoryBillingService applicantHistoryBillingService;
+
+    @GetMapping
+    @Operation(description = "Extract applicant history based on MAAT references.")
+    @StandardApiResponseCodes
+    public ResponseEntity<Object> getApplicantHistory() {
```

Comment:
Looking at Liam's PR I'm now wondering if we want to make the return type from this method `ResponseEntity<List<ApplicantHistoryBillingDTO>>`?

Reply:
Think this was left over from when I was looking at exception handling. Agree though this can be changed to the above now.